CHANGELOG.md

```diff
@@ -2,18 +2,22 @@
 
 ## Unreleased
 
-## [v1.4.1](https://github.com/coder/vscode-coder/releases/tag/v1.3.9) (2025-02-19)
+## [v1.4.2](https://github.com/coder/vscode-coder/releases/tag/v1.4.2)
```

Comment:
In terms of release flow we keep these under Unreleased until we're ready to make a separate tagged commit to cut the release :) We want to update that in the future so releasing is slightly less of a hassle, but for now would you mind sticking this under Unreleased?

Reply:
Done